### ISSUE TITLE
Sort out correct shebang for python

### ISSUE THREAD

**Issue by rifel123** [VirtusLab/git-machete#190]
Right now, while building container for rpm package, ambiguous shebang breaks build:
<img width="1124" alt="Screenshot 2021-08-25 at 09 13 54" src="https://user-images.githubusercontent.com/73847352/130795259-49462418-fe84-4e6a-a22a-a9002d5b48b8.png">

Clarification here is needed

**Comment by rifel123** [comment]
also please take a look into conversation in PR #182

**Comment by PawelLipski** [comment]
(https://github.com/VirtusLab/git-machete/pull/182#discussion_r694857623 specifically)

**Comment by PawelLipski** [comment]
Whatever the resolution, consider enforcing the practice via either `tox` if easily doable, or via a script under `ci/checks`

**Comment by rifel123** [comment]
Since support of python 2 is already dropped in git-machete, we should stick to use `python3` in shebangs. From [PEP394](https://www.python.org/dev/peps/pep-0394/#recommendation):
```
When packaging third party Python scripts, distributors are encouraged to change less specific shebangs to more specific ones. This ensures software is used with the latest version of Python available, and it can remove a dependency on Python 2. The details on what specifics to set are left to the distributors; though. Example specifics could include:

-Changing python shebangs to python3 when Python 3.x is supported.
-Changing python shebangs to python2 when Python 3.x is not yet supported.
-Changing python3 shebangs to python3.8 if the software is built with Python 3.8.
```
and further:
```
In cases where the script is expected to be executed outside virtual environments, developers will need to be aware of the following discrepancies across platforms and installation methods:
-Older Linux distributions will provide a python command that refers to Python 2, and will likely not provide a python2 
 command.
-Some newer Linux distributions will provide a python command that refers to Python 3.
-Some Linux distributions will not provide a python command at all by default, but will provide a python3 command by default.
```

Also, shebang should be removed from `github.py`, cause this file is not ment to be run as executable script.

**Comment by PawelLipski** [comment]
```
-Some newer Linux distributions will provide a python command that refers to Python 3.
-Some Linux distributions will not provide a python command at all by default, but will provide a python3 command by default.
```

Could you make a quick research how much is "some" for either case? easiest would be probably to check the latest Docker images for leading Linux distros, like `ubuntu`, `debian`, `fedora` etc.

**Comment by rifel123** [comment]
Ubuntu 20.04: `python3` provided, but not `python`,
Fedora: `python3` provided, but not `python`,
debian:  `python3` available only after update python, `python` points to python 2 anyway
mint: `python3` provided, but not `python`,
manjaro: Both `python3` and `python` available
elementary OS: `python3` provided, but not `python`,
win10: `python` points to python3 but `python3` opens microsoft store :) - but i have only python 3.7-9 installed
macOS: `python3` points to python3.X, `python2` points to python2.7

**Comment by PawelLipski** [comment]
Wow that's pretty comprehensive, nice... so it looks we should enforce `python3` in the shebangs then 🤔

**Comment by rifel123** [comment]
closed with #257
